slack.go

```diff
@@ -7,7 +7,7 @@ import (
 
 	slack "github.com/ashwanthkumar/slack-go-webhook"
 )
-
+var notifications = make(map[string]int)
```

Comment:
Move to the bottom of the file, with some whitespace around it, rather than crowding the unrelated comment on the line below.